stable_baselines3/common/evaluation.py

```diff
@@ -4,12 +4,12 @@
 import gym
 import numpy as np
 
-from stable_baselines3.common import base_class
+from stable_baselines3.common import type_aliases
 from stable_baselines3.common.vec_env import DummyVecEnv, VecEnv, VecMonitor, is_vecenv_wrapped
 
 
 def evaluate_policy(
-    model: "base_class.BaseAlgorithm",
+    model: "type_aliases.PolicyPredictor",
```

Comment:
What do you think about that? This would allow not to break the typing consistency for the variable named "model".

```suggestion
    model_or_policy: "type_aliases.PolicyPredictor",
```

Reply:
Maybe make this positional-only using `/`.

Reply:
not a big fan of `model_or_policy`, we could use `predictor` but I don't like much neither...

> Maybe make this positional-only using /.

why not, but unsure if needed

Reply:
> Maybe make this positional-only using `/`.

I don't think this is a good solution. My main concern here is that the term "model" refers everywhere else to an instance of BaseAlgorithm. Making it only positional does not solve the problem.
But if you think it's okay, we can leave it like that.

Reply:
> My main concern here is that the term "model" refers everywhere else to an instance of BaseAlgorithm

I understand your point, and in practice, it is the case in all examples except one in the doc (when we show the use of `model.policy`).
So, I would say it's fine.